abstract_egu.md

```diff
@@ -11,16 +11,24 @@ bibliography: references.bib
 ---
 
 R is probably the most important statistical computing language in academia.
-With almost 200 packages, it allows also the reproducible analysis of geographic data (see https://cran.r-project.org/web/views/Spatial.html and @bivand_applied_2013) which makes it a very attractive choice for persons working in all disciplines of the Earth, planetary and space sciences and a need for geographic data science.
+With almost 2000 packages it has been extended in many directions, including support for geospatial data [see https://cran.r-project.org/web/views/Spatial.html and @bivand_applied_2013].
```

Comment:
Should be more than 10,000!

Reply:
mmh, ok, this is misleading, I was thinking of R spatial packages, that's why I inserted the link to the Spatial task force

Reply:
but I like your suggestion, so I take this, and will update a bit